pkg/cache/internal/cache_reader.go

```diff
@@ -57,11 +57,11 @@ type CacheReader struct {
 }
```

Comment:
longer term we need to implement our own kcp-specific cache reader and plumb it through